src/meshes/t8code_mesh.jl

```diff
@@ -294,13 +361,41 @@ conforming mesh from a `p4est_connectivity` data structure.
 - `initial_refinement_level::Integer`: refine the mesh uniformly to this level before the simulation starts.
 """
 function T8codeMesh{NDIMS}(conn::Ptr{p4est_connectivity}; kwargs...) where {NDIMS}
```

Comment:
It's not really required to pass `NDIMS` as parameter since it can be deduced from the type of `conn` here and in 3D, is it?

Reply:
Agree. Adapted accordingly.